src/codeu/chat/client/core/Controller.java

```diff
@@ -120,6 +119,19 @@ public ConversationHeader newConversation(String title, Uuid owner, UserType def
     return response;
   }
 
+  public void removeConversation(ConversationHeader conversation){ 
+    ConversationHeader response = null; 
+    try(final Connection connection = source.connect()){
+
+      Serializers.INTEGER.write(connection.out(), NetworkCode.REMOVE_CONVERSATION_REQUEST);
+      Serializers.NULLABLE(ConversationHeader.SERIALIZER).write(connection.out(), conversation);
+    }catch(Exception ex){
+      LOG.error(ex, "Exception during call on server.");
+    }
+        
+  }
+
```

Comment:
Remove line breaks causing empty lines in 125, 131, and 133. Empty lines should make sense semantically.